typescript-client/api-reference/api-reference.mdx

```diff
@@ -66,7 +66,61 @@ The unique identifier of the step to delete.
 
 A `Promise` that resolves to the ID of the deleted step.
 
-### uploadFile()
+### createAttachment()
+
+> **createAttachment**(`params`): `Promise`\<`object`\>
+
+Uploads a file to Literal AI, and creates an attachment out of it. If called within a step wrapper, the attachment will be automatically added to the step.
+
+This method supports uploading either through direct content or via a file path. It first signs the upload through a pre-configured endpoint and then proceeds to upload the file using the signed URL.
```

Comment:
I didn't understand the signing part of the URL. I can see that it means we're secure.

Reply:
honestly it's not that important a point for the end user so i removed it and replaced it with a vague statement : "securely uploads a file to Literal AI ...".